datalog/src/main/native/include/wpi/DataLog.h

```diff
@@ -19,15 +19,15 @@
 #include <vector>
```

Comment:
I think as we're moving these headers, we should put them in either wpi/log or wpi/datalog instead of the top level wpi.  wpi/log would match the namespace, but datalog would be more consistent with struct and protobuf.  This would avoid the risk of conflicts with wpiutil since they're in two separate source directories?

Reply:
Should the namespace be changed to match as well?

Reply:
datalog is kind of a long name for a namespace.  I think leaving it as wpi::log is fine.